### ISSUE TITLE
gradle 'runIdea' needs to activate license

### ISSUE THREAD

**Issue by wezhang** [Microsoft/azure-tools-for-java#475]
![image](https://cloud.githubusercontent.com/assets/7385667/26229980/2d9282a2-3c78-11e7-837a-8961e3670637.png)

I'd like to debug plugin development.

I suppose community version is enough to me.

**Comment by wezhang** [comment]
I maded the following changes, it uses community version now:

```patch
diff --git a/PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties b/PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties
index 55445c3..7ff43a8 100644
--- a/PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties
+++ b/PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties
@@ -1,4 +1,4 @@
 javaVersion=1.8
 sources=false
-intellij_version=IU-2017.1
+intellij_version=IC-2017.1
 dep_plugins=org.intellij.scala:2017.1.15
\ No newline at end of file
```

**Comment by ziyel** [comment]
Yes. We can change the gradle.properties` to use diff version of intellij in local debug.

**Comment by wezhang** [comment]
Nice to have!

**Comment by wezhang** [comment]
Fixed by becc66c2184734aca73a2ca7fd2040ce5956d8f1
